components/InventoryItemListItem/index.tsx

```diff
@@ -121,13 +112,17 @@ export default function InventoryItemListItem({
       </TableCell>
       <TableCell
         sx={{
-          display: 'flex',
           alignItems: 'center',
-          width: '200px',
           wordBreak: 'break-word',
         }}
       >
-        {inventoryItem.assignee ? inventoryItem.assignee.name : ''}
+        {typeof inventoryItem.assignee === 'string'
```

Comment:
now that you've merged with main and have the new types, this whole block can be shortened to `inventoryItem.assignee ? inventoryItem.assignee.name : ''`